clusterDeployer.py

```diff
@@ -545,6 +692,14 @@ def update_etc_hosts(self) -> None:
         lh = host.LocalHost()
         lh.run("systemctl restart libvirtd")
 
+    def update_dnsmasq(self, *, setup: bool = True) -> None:
```

Comment:
A bit nit-picky but this is a "Host" property.  I think both `update_dnsmasq()` and `update_etc_hosts()` should be methods of `class ClusterHost` now that we're touching this code.  Class `ClusterHost` already has a `hostconn` member that can be used to interact with the host.  I'm not saying we need it now, but like that we can, in the future, configure dnsmasq on remote hosts too.

From `ClusterDeployer` we can access the local host through `self._local_host.update_etc_hosts()/self.local_host.update_dnsmasq()`.

Reply:
Sure, fine by me.  Let's keep them here for now.